packages/react/src/component-tests/IcLink/IcLink.cy.tsx

```diff
@@ -0,0 +1,100 @@
+/* eslint-disable react/jsx-no-bind */
+/// <reference types="Cypress" />
+
+import React from "react";
+import { IcLink } from "../../components";
+import { mount } from "cypress/react";
+
+const DEFAULT_TEST_THRESHOLD = 0.05;
+
+describe("IcHero e2e, A11y and visual regression tests", () => {
+  beforeEach(() => {
+    cy.injectAxe();
+  });
+
+  afterEach(() => {
+    cy.task("generateReport");
+  });
+
+  it("IcLink", () => {
+    mount(
+      <div style={{ margin: "16px" }}>
+        <IcLink href="/components/link/code">About our coffees</IcLink>
+      </div>
+    );
+    cy.checkHydrated("ic-link");
+    cy.compareSnapshot("defaultLink", DEFAULT_TEST_THRESHOLD + 0.05);
+    cy.get("ic-link").shadow().find("a").focus();
+    cy.compareSnapshot("defaul-focus", DEFAULT_TEST_THRESHOLD + 0.05);
```

Comment:
typo here - should be "default-focus"
 Also, you've used camelCase in the first screenshot name and kebab-case in this one (and screenshot naming is inconsistent elsewhere in the file)

Reply:
I opened a ticket to look at this across all cypress files (https://github.com/mi6/ic-ui-kit/issues/1523), so hopefully we can come up with a naming convention so that this doesn't happen 😊